src/rsz/include/rsz/Resizer.hh

```diff
@@ -639,7 +640,8 @@ protected:
   InstanceSeq inserted_buffers_;
   InstanceSet inserted_buffer_set_;
   Map<Instance *, std::tuple<LibertyPort *, LibertyPort *>> swapped_pins_;
-  Map<Instance *, Instance *> cloned_gates_;
+  std::stack<std::tuple<Instance *, Instance *>> cloned_gates_;
```

Comment:
Could give a using name to std::tuple<Instance *, Instance *>. I feel that it would help readability here.